test/functional/apps/management/_runtime_fields.js

```diff
@@ -23,13 +22,14 @@ export default function ({ getService, getPageObjects }) {
 
     before(async function () {
       await browser.setWindowSize(1200, 800);
-      await esArchiver.load('test/functional/fixtures/es_archiver/discover');
+      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
       // delete .kibana index and then wait for Kibana to re-create it
       await kibanaServer.uiSettings.replace({});
       await kibanaServer.uiSettings.update({});
     });
 
     after(async function afterAll() {
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover');
       await PageObjects.settings.navigateTo();
       await PageObjects.settings.clickKibanaIndexPatterns();
       await PageObjects.settings.removeLogstashIndexPatternIfExist();
```

Comment:
I don't think we need these last 3 statements.  The logstash index pattern should be loaded by the kibanaServer.importExport.load and should be removed by the kibanaServer.importExport.unload so I think this is just wasting time.